src/main.ts

```diff
@@ -2,4 +2,5 @@ export * from './components'
 export * from './compositions'
 export * from './errors'
 export * from './services'
-export * from './types'
\ No newline at end of file
+export * from './types'
+export { isRoute } from './guards/routes'
```

Comment:
I'd like to start doing this in the main.ts file. That way we have to be more intentional about what is exported publicly.